model/common/src/icon4py/model/common/grid/grid_manager.py

```diff
@@ -63,7 +63,10 @@ def __call__(self, array: data_alloc.NDArray):
         )
 
 
-CoordinateDict: TypeAlias = dict[gtx.Dimension, dict[Literal["lat", "lon"], gtx.Field]]
+# TODO(msimberg): x, y, z added temporarily for torus grids, but should they be in a separate dict?
+CoordinateDict: TypeAlias = dict[
+    gtx.Dimension, dict[Literal["lat", "lon", "x", "y", "z"], gtx.Field]
+]
```

Comment:
I haven't tried it yet, but I think the places where `CoordinateDict` is used could be changed to a `GeometryDict`. We possibly lose a bit of type safety (any `GeometryName` could be used instead of the closed set of naems currently in `CoordinateDict`).

@halungge do you have thoughts on what you'd prefer to see here (I see the TODO below, so you may already have thought about this)?